resources/scripts/profile

```diff
@@ -0,0 +1 @@
+export CONTAINERD_ADDRESS=/run/k3s/containerd/containerd.sock
```

Comment:
Users may expect `export CONTAINERD_NAMESPACE=k8s.io` too, if they expect `nerdctl ps` to show k3s containers.

Reply:
I believe this is supposed to be used for native container development à la docker-compose, in which case keeping images separate would be a feature.

I would keep it separate for now, and maybe later add a checkbox in the GUI to set the default namespace?

Reply:
SGTM

Reply:
@AkihiroSuda Thanks for checking in on this. I had thought about the namespace part. But, we want to expose full nerdctl to people who just want to work with containers. We will document using the Kubernetes namespace for those who want it for that.

Reply:
Should this be in `src/assets/scripts/…` instead?

Do we need this as a separate file, or is putting it inside of the `nerdctl` script enough (as in, `/usr/bin/env CONTAINERD_ADDRESS=… nerdctl …`)? Having a separate script makes more sense if we expect other things to need it, but I don't believe we expect users to manually run `limactl shell` (except as a troubleshooting step under our direction).

Reply:
I've set it up this way because I've been using nerdctl within the VM for troubleshooting. This just makes it easier.

Reply:
We will soon be able to put the thing into `lima.yaml` as

```yaml
env:
 CONTAINERD_ADDRESS=/run/k3s/containerd/containerd.sock
```

and it will automatically be added to `/etc/environment` and be available everywhere in the guest.

This doesn't work right now because our Alpine image uses an `sshd` that has been compiled without `pam` support (and also there is no `sshd.pamd` file either that enables `pam_env` for ssh sessions. Will work on this soonish, but for now using `.profile` seems fine.